stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
+# You should not edit this file by hand.
+# For more information, please see the documentation at:
+#   https://docs.haskellstack.org/en/stable/lock_files
+
+packages: []
+snapshots:
+- completed:
+    size: 535263
+    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/9/10.yaml
+    sha256: ba6ba39d048b5a2054142209f00e7e456e30081c03ea72b6135983633b234535
+  original: lts-9.10
```

Comment:
Also, is this necessary? Seems like something that was produced on a local run

Reply:
> Also, is this necessary? Seems like something that was produced on a local run

Good point, removed.